static/settings.js

```diff
@@ -124,6 +126,40 @@ define(function (require) {
         });
         add(root.find('.hoverShowSource'), 'hoverShowSource', true, Checkbox);
         add(root.find('.hoverShowAsmDoc'), 'hoverShowAsmDoc', true, Checkbox);
+        var themeSelect = root.find('.theme');
+        add(themeSelect, 'theme', themes.default.id, Select,
+            _.map(themes, function (theme) {
+                return {label: theme.id, desc: theme.name};
+            })
+        );
+
+        function handleThemes() {
+            if (!colourSchemeSelect || !themeSelect) return;
+            var newTheme = themeSelect.val();
+            // Store the scheme of the old theme
+            $.data(themeSelect, 'theme-' + $.data(themeSelect, 'last-theme'), colourSchemeSelect.val());
```

Comment:
Can you add a dependency on `$` here - at the top of the file with the other requires put `var $ = require('jquery');` or similar. (It works at the moment but only because jquery pollutes the global namespace)